network_observability/network-observability-overview.adoc

```diff
@@ -25,33 +25,6 @@ The Network Observability Operator provides the Flow Collector API custom resour
 
 {product-title} console integration offers overview, topology view and traffic flow tables.
```

Comment:
Just checking if this "overview" mention here needs to be adjusted with this section below being removed?

Reply:
yeah I think initially I removed this Overview mention totally in favor of a dedicated assembly, but I actually think we should mention it briefly and link to the new assemblies. Thanks for pointing this out!